pkgs/development/python-modules/torch/bin.nix

```diff
@@ -74,7 +93,9 @@ in buildPythonPackage {
     # Includes CUDA and Intel MKL, but redistributions of the binary are not limited.
     # https://docs.nvidia.com/cuda/eula/index.html
     # https://www.intel.com/content/www/us/en/developer/articles/license/onemkl-license-faq.html
-    license = licenses.bsd3;
+    # torch's license is BSD3.
+    # torch-bin includes CUDA and MKL binaries, therefore unfreeRedistributable is set.
+    license = licenses.unfreeRedistributable;
```

Comment:
Changed the license of torch-bin to unfreeRedistributable.

Reply:
@SomeoneSerge 
I've created a sparate git commit to update the license  to unfreeRedistributable.

Reply:
@SomeoneSerge 
I've created a sparate git commit to update the license  to unfreeRedistributable.

Reply:
The new comment overlaps with the old one, we need to merge them. Thoughts:

- [👍🏻] You already explain that Pytorch is redistributed under BSD3
- [✅] Keep the links to CUDA EULA and Intel Open Source License
- [ ] When mentioning Intel Open Source License, also include the link to the short identifier: https://spdx.org/licenses/Intel.html, (kudos @fabaff)
- [ ] Add a link to `lib.licenses.issl` https://www.intel.com/content/dam/develop/external/us/en/documents/pdf/intel-simplified-software-license.pdf
- [ ] Explain that Intel's [oneAPI](https://github.com/oneapi-src/oneMKL) and [mkl-dnn](https://github.com/Intel-tensorflow/mkl-dnn) are free ASL20
- [ ] Explain that upstream wheels link Pytorch statically against MKL (which are `lib.licenses.issl` unfree and redistributable)
- [ ] Explain that upstream wheels include cuda applications, subject to CUDA EULA
- [ ] Explain that since the whole thing is distributed as a single package, we have to mark the final derivation as `unfreeRedistributable`

I actually didn't notice any components that refer to the Intel Opensource License, but we should keep the links for reference